lib/data.js

```diff
@@ -0,0 +1,18 @@
+export const titles = [
+  {
+    date: "22.12.2022",
+    entrytitle: "Entry 1",
```

Comment:
you can camel case those keys: 
```suggestion
    entryTitle: "Entry 1",
```

Reply:
You should  camelcase your keys here